src/main/resources/public/css/styles.css

```diff
@@ -14,6 +14,54 @@ th, td {
 /***********************
 *INDIVIDUAL VARIABLES
 ***********************/
+#easyHard{
+	padding-top:30px;
+	width:100%;
+	height:140%;
+	position:absolute;
+	background-color:rgba(255, 255, 255, 0.8);
+}
+#easyHard > #easyDiv, #hardDiv{
+	margin-top:100px;

```

Comment:
This might be a little too much margins from the top. However it may look different on my computer than yours.